docs/user_guide/HOWTO/Alternate_SIMP_Config_Environment.rst

```diff
@@ -0,0 +1,27 @@
+.. _howto-use-an-alternate-simp-config-environment:
+
+HOWTO Use an Alternate ``simp config`` Environment
+==================================================
+
+Generally, when running the ``simp config`` and ``simp bootstrap`` commands, all
+items will be placed into the ``production`` :term:`Puppet Environment` and
+:term:`SIMP Secondary Environment`.
+
+If you are installing SIMP onto a preexisting system, you may want to instead
+install into an environment other than ``production``.
+
+As of version 6.0.0, the ``simp-cli`` was updated to support setting an
+alternate initial environment using a shell environment variable called
+``SIMP_ENVIRONMENT``.
```

Comment:
The 6.0.0 here refers to the version of `simp-cli`, but in the context of the SIMP docs, it's more likely to be interpreted as a SIMP release.  Here's one way to clarify it:

```suggestion
As of ``simp-cli`` version 6.0.0 (new with SIMP 6.5), you can configure
the ``simp config`` and ``simp bootstrap`` commands to use an
alternate initial environment by setting the shell environment 
variable, ``$SIMP_ENVIRONMENT``.
```